assets/scss/_description.scss

```diff
@@ -0,0 +1,51 @@
+.description {
+  position: relative;
+  margin-top: 40px;
+  padding-bottom: 150px;
+
+  &__background {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: -2;
+  }
+
+  &__title {
+    padding-top: 100px;
+
+    &::after {
+      content: "";
+      display: block;
+      padding-top: 7px;
+      border-bottom: 3px solid #b4437b;
+      @include media-queries("desktop") {
+        width: 350px;
+        margin-left: 100px;
+      }
+    }
+  }
+
+  &__text {
+    font-family: "Montserrat", sans-serif;
+    line-height: 1.5rem;
+    padding: 5px 0;
+    font-size: 12px;
+    @include media-queries("desktop") {
+      padding: 10px 300px 10px 0;
+      font-size: 16px;
+    }
+  }
+
+  &__graphics {
+    padding: 20px 0 30px;
+    z-index: -1;
```

Comment:
z-index won't work without setting some position which is different from default - `static`